src/main/java/com/programmers/voucher/domain/VoucherFactory.java

```diff
@@ -0,0 +1,16 @@
+package com.programmers.voucher.domain;
+
+import com.programmers.voucher.dto.request.VoucherCreationRequest;
+
+import java.util.UUID;
+
+public class VoucherFactory {
+    public static Voucher createVoucher(VoucherCreationRequest voucherCreationRequest) {
+        VoucherType voucherType = VoucherType.valueOf(voucherCreationRequest.voucherType());
```

Comment:
[nit]
```suggestion
        String requestVoucherType = voucherCreationRequest.voucherType().toUpperCase();
        VoucherType voucherType = VoucherType.valueOf(requestVoucherType);
```
가독성 + 디버깅할 때 값 체크? + UpperCase 처리해주기?(valueOf는 caseSensitive!)